database/integration_test.go

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) 2023 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package database
+
+import (
+	"os"
+	"reflect"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/go-vela/server/database/worker"
+
+	"github.com/go-vela/types/library"
+)
+
+func TestDatabase_Integration(t *testing.T) {
+	// check if we should skip the integration test
+	if os.Getenv("INTEGRATION") == "" {
```

Comment:
Is this a common way to skip a set of tests by default with go tests? Is `INTEGRATION` a common will known env flag? Or is there a native way to "mark"/"tag" tests with go?

Reply:
This article should help - https://konradreiche.com/blog/how-to-separate-integration-tests-in-go

The name of the environment variable is arbitrary assuming the `Makefile` reflects the same.